x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/apikey/RestGrantApiKeyAction.java

```diff
@@ -86,35 +117,29 @@ public String getName() {
         return "xpack_security_grant_api_key";
     }
 
-    public static GrantApiKeyRequest fromXContent(XContentParser parser) throws IOException {
```

Comment:
This is only used in tests, so I don't think it's too hacky to move it to the RequestTranslator class. I think it's still a useful utility method to keep so I'd prefer not to remove it (even though it's currently not used in prod code).